src/api/verification_library.py

```diff
@@ -100,3 +111,79 @@ def get_library_item(self, item_name: str) -> Dict:
         }
 
         return item_dict
+
+    def validate_library(self, items: List[str] = []) -> pd.DataFrame:
+        """Check the validity of library items definition. This validity check includes checking the completeness of json specification (against library json schema) and Python verification class definition (against library class interface) and the match between the json and python implementation.
+
+        Args:
+            items: list of str, default []. Library items to validate. By default, summarize all library items loaded at instantiation.
+
+        Returns:
+            pandas.DataFrame that contains validity information of library items.
+        """
+
+        # check `items` type
+        if not isinstance(items, list):
+            logging.error(f"items needs to be list. It cannot be a {type(items)}.")
+            return None
```

Comment:
Do we want to provide also input validation here if the list of items is empty?

Reply:
Addressed.